### PR TITLE
Remove "message" / "msg" from endpoints #215

### PR DESCRIPTION
**Pull request by halsing**
I removed "message" from endpoint because marshmallow use "msg". 
I also changed few corresponding test. 

To test my changes You should use "make test" and if frontend use "message", You have to change in to "msg". 

I found one endpoint which use "message": If you make GET request in url "/auth/refresh/", but this is not our "message" but probably from marshmallow or somewhere else.

### CODE REVIEW

File: backend/config.py [hunk 1]
```diff
@@ -17,15 +17,18 @@ class DevelopmentConfig(Config):
...
+
+    SQLALCHEMY_DATABASE_URI = (
+        f"postgresql://{DB_USER}:{DB_PASSWORD}@{DB_HOSTNAME}:{DB_PORT}/{DB_NAME}"
     )
     SQLALCHEMY_ENGINE_OPTIONS = {"pool_recycle": 299}
     SQLALCHEMY_TRACK_MODIFICATIONS = False
     SECRET_KEY = os.environ.get("SECRET_KEY")
 
     JWT_BLACKLIST_ENABLED = True
+    JWT_ERROR_MESSAGE_KEY = "msg"
```
Comment (Maciek246) [inline backend/config.py]: Is this required? From documentation `flask-jwt-extended`:

> Defaults to 'msg'.

Is there any reason?

**Comment by Maciek246** [comment]
Some changes is duplicated here https://github.com/CodeForPoznan/codeforpoznan.pl_v3/pull/238

**Comment by w1stler** [comment]
@halsing please resolve conflicts here.

**Comment by OtisRed** [comment]
@stanislawK has doubts towards this and it hangs here for too long so I decided to close it

**Pull request closed without merging**